submissions/solaryasha/html-css-popup/index.html

```diff
@@ -0,0 +1,224 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>HTML-CSS-Popup</title>
+</head>
+<body>
+    <nav class="navigation">
+        <ul class="nav_menu">
+            <li class="menu_card mail_button">
+                <a href="#">Mail</a>
+            </li>
+            <li class="menu_card image_button">
+                <a href="#">Images</a>
+            </li>
+            
+            <li class="menu_card apps_button">
+                
+                <input id="btn" class="menu-btn" type="checkbox">
```

Comment:
Add styles on `:hover` state for popup icon.

Reply:
Done!